File: python/0-count-performance/count.py
import time

def main():
  start = time.time()
  count = 0
  for i in range(0,1000000000):
    count += 1
  end = time.time()
  elapsedTime = end-start
  print(f'Count is {count}.')
  print(f'Execution time {elapsedTime} seconds.')

if __name__ == '__main__':
  main()

File: javascript/0-count-performance/count.js
function main(){
  let count = 0;
  const start = Date.now();
  for(let i = 0; i < 1000000000; i++){
    count += 1;
  }
  const end = Date.now();
  const elapsedTime = (end - start) / 1000.0;
  console.log(`Count is ${count}`);
  console.log(`Execution time ${elapsedTime} seconds`);
}

main();